### PR TITLE
cronie_common: Increase pointer assignment check

### PR DESCRIPTION
**Pull request by likunyur**

### CODE REVIEW

File: cronie_common.c [hunk 1]
```diff
@@ -107,6 +107,9 @@ int expand_envvar(const char *source, char *result, size_t max_size) {
         }
 
         envvar_name = malloc(envvar_name_size + 1);
+	if (!envvar_name)
```
Comment (t8m) [inline cronie_common.c:110]: Please use `envvar_name == NULL`

Comment (likunyur) [reply cronie_common.c]: Thank you. I resubmitted the patch.

File: cronie_common.c [hunk 2]
```diff
@@ -107,6 +107,9 @@ int expand_envvar(const char *source, char *result, size_t max_size) {
         }
 
         envvar_name = malloc(envvar_name_size + 1);
+	if (envvar_name == NULL)
+		goto too_big;
+
```
Comment (t8m) [inline cronie_common.c:112]: One more nit - could you please replace tabs with spaces? Also the goto should be then aligned with the condition (i.e., 4 spaces indentation per level)

Comment (likunyur) [reply cronie_common.c]: I resubmitted the patch.

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Merged. Thank you for your contribution!

**Pull request merged**